apps/documentation/src/routes/documentation/tutorial/redirections.mdx

```diff
@@ -13,13 +13,11 @@ import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
 # Redirections
 
-## Handle redirections
+What if there is a Pokémon among all of them that should be considered the GOAT? What
+we are going to do right now is create a new route `/pokemons/GOAT` that points to the best
+Pokémon of the first generation.
 
-What if there is a pokemon among all of them that should be considered the GOAT? What
-we are going to do right now is creating a new route `/pokemons/GOAT` that points to the best
-pokemon of the first generation.
-
-First let's create a new route by just creating an new file `/pokemons/GOAT.rs` and pasting the following code:
+First, let's create a new route by just creating a new file `/pokemons/GOAT.rs` and pasting the following code:
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The plural of ‘Pokémon’ is “Pokémon”. (POKEMONS[1])
Suggestions: `Pokémon`
URL: https://en.wikipedia.org/wiki/Pok%C3%A9mon#Name 
Rule: https://community.languagetool.org/rule/show/POKEMONS?lang=en-US&subId=1
Category: GRAMMAR
<!-- __reviewdog__:ChA1ZTI2Njc4NGM2ZjA3NTAxEgxMYW5ndWFnZVRvb2w= -->